plugin/src/sbt-test/sbtorgpolicies/ci-after-succes-cmd/project/build.properties

```diff
@@ -0,0 +1 @@
+sbt.version=1.2.7
```

Comment:
Could we avoid adding `build.properties` files in scripted tests?

Reply:
Sure, I'll remove it. Let's see if there are suggestions from @franciscodr also.

Reply:
Sure, I'll remove it. Let's see if there are suggestions from @franciscodr also.